docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -674,8 +684,8 @@ for tips on diagnosing and preventing them.
 [[task-queue-backlog]]
 === Task queue backlog
 
-A backlogged task queue can prevent tasks from completing and 
-put the cluster into an unhealthy state. 
+A backlogged task queue can prevent tasks from completing and
+put the cluster into an unhealthy state.
```

Comment:
Are these format changes something you did or they happened "accidentally". I am pointing this out because I am not sure if this was intentional. In my case intellij was making similar changes and it blew up my PR since I touched a lot of doc files. So keep that in mind.